dali/dafilesrv/dafilesrv.cpp

```diff
@@ -691,6 +692,8 @@ int main(int argc,char **argv)
     } perfHook;
     startPerformanceMonitor(10*60*1000, PerfMonStandard, &perfHook);
     writeSentinelFile(sentinelFile);
+
+    setKeyIndexCacheSize(0); // disable IKeyIndex caching
```

Comment:
Need to discuss.
If enabled, or if not cleared - key writes will not be reflected, i.e. remote key reads will pick up old copies.